app/controllers/api/v1/teachers_controller.rb

```diff
@@ -10,11 +10,16 @@ def show
       end
 
       def create
-        Teacher.create(teacher_params)
-        render json: TeacherSerializer.new(Teacher.last)
+        # Teacher.create(teacher_params)
+        # render json: TeacherSerializer.new(Teacher.last)
```

Comment:
Same question. Are we leaving this for a reason?

Reply:
They are gone now!